module/build.sbt

```diff
@@ -5,17 +5,19 @@ name               := "play-googleauth"
 
 organization       := "com.gu"
 
-scalaVersion       := "2.11.8"
+scalaVersion       := "2.12.2"
```

Comment:
I definitely want to move to `2.12` to be the default, but changing the example to match is a good call.